tools/doc/package.json

```diff
@@ -16,7 +16,8 @@
     "unified": "^7.0.0",
     "unist-util-find": "^1.0.1",
     "unist-util-select": "^1.5.0",
-    "unist-util-visit": "^1.4.0"
+    "unist-util-visit": "^1.4.0",
+    "highlightjs": "^9.10.0"
```

Comment:
I'm surprised `npm` didn't insert this in alphabetical order. Are you using an old version of `npm` possibly?

Reply:
I think this is my fault from doing a rebase/merge. I'll put it in the right place and re-push.

Reply:
Ah, a rebase, that makes sense. It doesn't much matter. I was just wondering how it happened.

Reply:
I went ahead and fixed it (and rebased again) -- that way if someone else adds a package they won't be confused having it move around.

Reply:
Probably an oversight, but the [`highlightjs` package](https://www.npmjs.com/package/highlightjs) is now deprecated, so you'd want the newer [`highlight.js` one](https://www.npmjs.com/package/highlight.js) instead.

Reply:
👍 Good catch! I went ahead and changed it to `highlight.js`. Thanks!